### DISCUSSION TITLE
Plain Binary for android is not available

### DISCUSSION THREAD

**Discussion by nikhere1** [LeelaChessZero/lc0#1714] [General]
Hello,

Does anyone know where I can get Lc0 plain binaries (without in-built weight files) for Android? I found links for armv7-a and aarch64 on Lc0 website https://lczero.org/dev/wiki/running-lc0-on-android-with-a-chess-gui/ But the links don't work and give the following error.

{"message":"Artifact not found or access denied."}

I couldn't find the official plain binaries on Internet myself. Is there a way to generate them myself?

**Comment by lealgo** [comment]
@nikhere1 

Hi,

I couldn't find the plain binaries in the release artifacts... But you could use the ones from the release instead, as they also work with weights other than the built-in. To do that just download the apk from the release page and rename it to zip, then open it with a regular archiver. Inside the zip, the engine is located at `/lib/arm64-v8a/liblc0.so` just copy and rename it to anything you like.

Regards,
Leandro

**Comment by nikhere1** [comment]
@lealgo 

Thank you so much for your reply. Yes. that is a possibility. Thank you for sharing.
